cli/src/plugins/javadoc/index.ts

```diff
@@ -323,6 +339,32 @@ function makeClassDocPageBody(args: MakeSectionArgs): Node[] {
           .flat(1),
     }),
 
+    ...makeSection({
+      ...args,
+      depth,
+      title: "Enum Constant Detail",
+      shouldMakeSection: () => doc.enumConstants.length !== 0,
+      makeBody: () =>
+        doc.enumConstants
+          .map((doc) => [
+            project.declareEntity({
+              canonicalName: doc.containingClass?.asString + "_" + doc.name,
+              pageUri,
+            }),
+            md.heading(3, md.inlineCode(doc.name)),
+            md.paragraph([
+              md.text(
+                doc.modifiers +
+                  " " +
+                  doc.containingClass?.typeName +
```

Comment:
In js, rather than concatenating with `+ " " +`, prefer string template literals:

```md.text(`${doc.modifiers} ${doc.containingClass?.typeName} ...`)```